src/sentry/workflow_engine/models/action_alertruletriggeraction.py

```diff
@@ -10,5 +15,5 @@ class ActionAlertRuleTriggerAction(DefaultFieldsModel):
 
     __relocation_scope__ = RelocationScope.Excluded
 
+    alert_rule_trigger_action_id = BoundedBigIntegerField(null=True)
```

Comment:
It looks like you're marking this nullable, is that intentional? I don't see any db operations in the generated sql

Reply:
oh good call, that I think I just copied from the other ones that are supposed to be nullable but this one isn't